ms-frontend/src/components/CreateReviewModal.vue

```diff
@@ -0,0 +1,226 @@
+                                    Please select the rating
+                                </div>
+                            </div>
+                        </div>
+                        <!-- Text area -->
+                        <div class="flex flex-col items-start gap-1 flex-[1_0_0] self-stretch">
+                            <Textarea v-model="reviewText" type="text" placeholder="Share details of your experience"
+                                class="self-stretch text-base not-italic font-light leading-[normal]" rows="4" />
+                            <!-- Error message for review text -->
+                            <div v-if="!textValid"
+                                class="text-error text-[10px] not-italic font-light leading-[normal] font-inter">
+                                Your text must contain more than 5 and less than 200 words
+                            </div>
+                        </div>
+
+                        <div class="text-medium-emphasis font-inter text-base not-italic font-light leading-[normal]">
+                            Images
+                        </div>
+                        <div class="relative w-full">
+                            <FileUpload v-model="reviewPicture"> </FileUpload>
+                        </div>
+                    </div>
+
+                    <!-- Divider -->
+                    <div class="h-px shrink-0 self-stretch opacity-60 bg-medium-emphasis"></div>
+
+
+                    <!-- Footer -->
+                    <div class="flex justify-end items-start self-stretch">
+
+                        <!-- Cancel button -->
+                        <div class="flex justify-center items-center  px-2 py-1">
+                            <Button outlined rounded label="Cancel" @click="$emit('closeModal')"></Button>
+                        </div>
+
+                        <!-- Post button -->
+                        <div class="flex justify-center items-center px-2 py-1">
+                            <Button rounded label="Post" :disabled="!checkInputValidity()"
+                                @click="createReview"></Button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+
+import Button from 'primevue/button'
+import BaseIcon from '@/icons/BaseIcon.vue'
+import { computed, ref, watch } from 'vue'
+import { useAuthStore } from '@/stores/auth'
+import FileUpload from '@/components/FileUpload.vue'
+import { reviewService } from '@/services/reviewService'
+import Textarea from 'primevue/textarea'
+import type { User } from '@/types/User'
+import type { Location } from '@types/Location'
+import type { Review } from '@types/Review'
+
+const authStore = useAuthStore()
+
+const signedInUser = authStore.user
+const myId = signedInUser?.id ?? ''
```

Comment:
```suggestion
import { ref, watch } from 'vue'
import FileUpload from '@/components/FileUpload.vue'
import { reviewService } from '@/services/reviewService'
import Textarea from 'primevue/textarea'
import type { Location } from '@types/Location'
import type { Review } from '@types/Review'

```

Remove unused imports.